### PR TITLE
456 fe content panel improvements in navigation

### PR DESCRIPTION
**Pull request by mitkapanarin**
This commit updates the Navigation component, introducing several new features and improvements:

1. Conditional Rendering:
   - Implemented logic to differentiate between public navigation and content panel navigation
   - The component now adapts its display based on the current route

2. Content Panel Navigation:
   - Added specific links for Blogs and Tags pages within the content panel area
   - Implemented conditional rendering to show these links only when in the content panel

3. User Authentication Features:
   - Integrated user avatar display for authenticated users
   - Implemented a dropdown menu accessible by clicking on the user avatar
   - Menu includes user information (name and email) and a sign-out option

4. Improved User Experience:
   - Added a click-outside listener to automatically close the user menu when clicking elsewhere on the page
   - Enhanced the visual feedback for active navigation links
   
   
image

### CODE REVIEW

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -6,7 +6,7 @@ jobs:
     timeout-minutes: 60
     runs-on: ubuntu-latest
     env:
-      BASE_URL: 'http://localhost:3000'
+      BASE_URL: ''
```
Comment (edichoska) [inline .github/workflows/playwright.yml:9]: why was this removed?

File: .gitleaks.toml
Comment (edichoska) [inline .gitleaks.toml:1]: I am confused by this change. Why was this necessary?

**Review by edichoska — CHANGES_REQUESTED** [review]
@mitkapanarin Please clean up this PR. I know you were working on the Invite Member ticket for the admin-panel, and I can see you transferred all that code to this branch as well.
This branch and the ticket that corresponds to it should be only for content panel and the navigation.
Rerequest review when you have changed this. 🙏

**Comment by edichoska** [comment]
@mitkapanarin should we close this PR, as I see you created a new one?

**Comment by mitkapanarin** [comment]
Yes, I created a new branch and made all the changes there. Is that okay?


Sent from Yahoo Mail for iPhone


On Sunday, December 8, 2024, 4:33 PM, Emilija Dichoska ***@***.***> wrote:




@mitkapanarin should we close this PR, as I see you created a new one?

**Comment by edichoska** [comment]
> Yes, I created a new branch and made all the changes there. Is that okay? Sent from Yahoo Mail for iPhone On Sunday, December 8, 2024, 4:33 PM, Emilija Dichoska ***@***.***> wrote: @mitkapanarin should we close this PR, as I see you created a new one? — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you were mentioned.Message ID: ***@***.***>

Yes, ofcourse. Just wanted to double check if I can close this PR.

**Comment by edichoska** [comment]
Duplicate

**Pull request closed without merging**